server/gitrest/package.json

```diff
@@ -48,7 +48,7 @@
 		"@fluid-tools/build-cli": "^0.56.0",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.56.0",
-		"@fluidframework/eslint-config-fluid": "^5.4.0",
+		"@fluidframework/eslint-config-fluid": "^5.7.4",
```

Comment:
Not sure if the build will complain (it should) but we should update the individual packages in server/gitrest/packages too to match this. Same for historian. That should dedupe some transitive dependencies in the lockfile.

Reply:
I can do that. They must have already been mismatched, or my find-and-replace strategy would have found them. But I'm happy to upgrade more.

Reply:
Updated. Note that the services packages were quite a ways behind, so there are a fair number of code changes to review. You have done this to yourself 😜

Reply:
Ah, we don't apply syncpack on any of the server release groups🥲 so the build wouldn't fail